File: src/IsbInterop.CSharp/EmptyIsbObject.cs
namespace IsbInterop
{
  /// <summary>
  /// Пустая объект IS-Builder.
  /// </summary>
  internal class EmptyIsbObject : IsbComObjectWrapper
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwObject">COM-объект.</param>
    public EmptyIsbObject(object rcwObject) : base(rcwObject) { }
  }
}

File: src/IsbInterop/EmptyIsbObject.vb
''' <summary>
''' Пустой объект.
''' </summary>
Friend Class EmptyIsbObject
    Inherits IsbComObjectWrapper

    ''' <summary>
    ''' Конструктор.
    ''' </summary>
    ''' <param name="rcwObject">COM-объект.</param>
    Public Sub New(rcwObject As Object)
        MyBase.New(rcwObject)
    End Sub
End Class